### PR TITLE
Worked Example: Easy releases

### PR DESCRIPTION
**Pull request by romifz**

### CODE REVIEW

File: standard/docs/en/getting_started/releases_and_records.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ In some cases, a publisher might only have the data infrastructure to publish an
 * ocds-a2ef3d01-1594121/2
 * ocds-a2ef3d01-1594121/3
 
+Other possible approaches are to appending the date or timestamp of the last data change to the `ocid` (if any is saved in the source system(s)) or to use a hash of the release data to generate an unique suffix. The [implementation guidance](../implementation/easy-releases.md) has detailed examples of both strategies.
 
 ### Records
```
Comment (mrshll1001) [inline standard/docs/en/getting_started/releases_and_records.md]: Change "to appending" to "to append".

**Review by mrshll1001 — CHANGES_REQUESTED** [review]
`standard/docs/en/examples/easy-releases/worked-example-1/4-contract.json` and `standard/docs/en/examples/easy-releases/worked-example-2/4-contract.json` need a Z on the end of releases/0/contracts/0/period/endDate according to the validator and there's a single typo in `standard/docs/en/getting_started/releases_and_records.md` which I've highlighted.

Otherwise this is seriously good. Nice work! I'll approve once those are fixed :-)

**Comment by romifz** [comment]
Thanks Matt! I made the changes requested.

**Review by mrshll1001 — APPROVED** [review]
All good :-)

**Comment by mrshll1001** [comment]
I added a paragraph on considerations for release tags to this, so it should be reviewed further.

**Comment by jpmckinney** [comment]
@romifz Can you review @mrshll1001's new changes, then request my review?

**Comment by romifz** [comment]
Hi @mrshll1001, thanks for the additions. I believe the release tags section actually applies to scenario 2, so I will modify the example.

File: standard/docs/en/getting_started/releases_and_records.md (hunk 1 above)
Comment (romifz) [reply standard/docs/en/getting_started/releases_and_records.md]: I'm removing this edit since the file is going to be deleted in #997.

**Comment by romifz** [comment]
Hi @mrshll1001, I've made some changes after reading the tag considerations. I believe that, when providing a single release per process, a tag for each section should always be provided as you indicated. Would you mind to review the changes again, and see if they make sense to you?

**Review by mrshll1001 — APPROVED** [review]
Cheers Romi, this all looks good. Sample data matches what we say in the text perfectly.

**Comment by jpmckinney** [comment]
Hi @romifz in future, please:

* Don't force push. It makes the review comments disappear, and it's harder to see what has changed since the last review. If you want a clean git history, you can just request that I use the "Squash and merge" option in the "Merge pull request" dropdown.
* Don't line wrap in Markdown files. It will cause diffs to be very large if we edit and re-wrap content in future. You can instead configure your editor to visually line wrap, without actually changing the whitespace in the file.

**Comment by jpmckinney** [comment]
I'm going to merge this into #997 and then merge to 1.1-dev, as the two pull requests depend on each other and can't be merged individually.

**Pull request merged**